### PR TITLE
Tiger delta

### PR DESCRIPTION
**Pull request by Rub21**
from https://github.com/osmlab/osmlint/issues/84

cc. @geohacker @batpad @maning @planemad

### CODE REVIEW

File: validators/tigerDelta/map.js [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+    newLine.push([
+      line[i].x * 4096 / extent,
+      line[i].y * 4096 / extent
+    ]);
+  }
+  return newLine;
+}
+
+function filterShort(line) {
+  return dist(line) >= 30; //line length is at least 30 pixels
```
Comment (geohacker) [inline validators/tigerDelta/map.js]: Hey @Rub21 - can you explain how the distance is in `pixels`? Can we use [turf.linedistance](turfjs.org/static/docs/module-turf_line-distance.html)?

Comment (Rub21) [inline validators/tigerDelta/map.js]: I've migrated from https://github.com/mapbox/road-diff/ , it works really faster. so i don't think is necessary use  turf.linedistance.

**Comment by Rub21** [comment]
@geohacker @batpad can you check here?  this is ready to merge in master.

File: validators/tigerDelta/map.js (hunk 1 above)
Comment (geohacker) [inline validators/tigerDelta/map.js]: @Rub21 interesting. I'll tag @mourner here if he can add thoughts, but we can merge this.

Comment (mourner) [inline validators/tigerDelta/map.js]: Yes, on this scale the difference between geographic distance and Euclidean is very small, but it's _much_ faster.

**Pull request merged**